keylime/db/verifier_db.py

```diff
@@ -29,7 +29,7 @@ class VerfierMain(Base):
     vtpm_policy = Column(String(1000))
     meta_data = Column(String(200))
     allowlist = Column(Text(429400000))
-    ima_sign_verification_keys = Column(String(1048576))
+    ima_sign_verification_keys = Column(Text(429400000))
```

Comment:
LGTM, but I am not sure if the length limit actually works, for what I saw, it's just a longtext in the database.